### PR TITLE
Move SE fragment to main screen

### PR DESCRIPTION
**Pull request by sharvaniharan**
**Phab**: https://phabricator.wikimedia.org/T231442

### CODE REVIEW

File: app/src/main/java/org/wikipedia/navtab/NavTab.java [hunk 1]
```diff
@@ -29,6 +30,11 @@
             return HistoryFragment.newInstance();
         }
     },
+    SE(R.string.nav_item_suggested_edits, R.drawable.ic_mode_edit_themed_24dp) {
```
Comment (cooltey) [inline app/src/main/java/org/wikipedia/navtab/NavTab.java]: Can we have the full name `SUGGESTED_EDITS` on it?

File: app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt [hunk 2]
```diff
@@ -181,35 +176,29 @@ class SuggestedEditsTasksFragment : Fragment() {
...
             addImageCaptionsTask.unlockMessageText = String.format(getString(R.string.suggested_edits_task_image_caption_edit_disable_text), targetForAddCaptions)
 
-            if (Prefs.isSuggestedEditsAddCaptionsUnlocked() && WikipediaApp.getInstance().language().appLanguageCodes.size >= MIN_LANGUAGES_TO_UNLOCK_TRANSLATION) {
+            if (WikipediaApp.getInstance().language().appLanguageCodes.size >= MIN_LANGUAGES_TO_UNLOCK_TRANSLATION) {
                 displayedTasks.add(translateImageCaptionsTask)
                 translateImageCaptionsTask.unlockMessageText = String.format(getString(R.string.suggested_edits_task_image_caption_edit_disable_text), targetForTranslateCaptions)
-                translateImageCaptionsTask.showActionLayout = !Prefs.isSuggestedEditsTranslateCaptionsUnlocked()
-                translateImageCaptionsTask.disabled = !Prefs.isSuggestedEditsTranslateCaptionsUnlocked()
+                translateImageCaptionsTask.showActionLayout = false
+                translateImageCaptionsTask.disabled = false
```
Comment (cooltey) [inline app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt]: These default values are `false`. No needs to set them as `false` again.

File: app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt [hunk 3]
```diff
@@ -181,35 +176,29 @@ class SuggestedEditsTasksFragment : Fragment() {
...
             }
 
             if (WikipediaApp.getInstance().language().appLanguageCodes.size < MIN_LANGUAGES_TO_UNLOCK_TRANSLATION && Prefs.showSuggestedEditsMultilingualTeaserTask()) {
                 displayedTasks.add(multilingualTeaserTask)
-                multilingualTeaserTask.showActionLayout = Prefs.isSuggestedEditsTranslateDescriptionsUnlocked()
-                multilingualTeaserTask.disabled = !Prefs.isSuggestedEditsTranslateDescriptionsUnlocked()
+                multilingualTeaserTask.showActionLayout = true
+                multilingualTeaserTask.disabled = false
```
Comment (cooltey) [inline app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt]: Same

**Comment by sharvaniharan** [comment]
Changes done @cooltey

File: app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt [hunk 4]
```diff
@@ -181,35 +176,25 @@ class SuggestedEditsTasksFragment : Fragment() {
...
+            if (WikipediaApp.getInstance().language().appLanguageCodes.size >= MIN_LANGUAGES_TO_UNLOCK_TRANSLATION) {
                 displayedTasks.add(translateImageCaptionsTask)
                 translateImageCaptionsTask.unlockMessageText = String.format(getString(R.string.suggested_edits_task_image_caption_edit_disable_text), targetForTranslateCaptions)
-                translateImageCaptionsTask.showActionLayout = !Prefs.isSuggestedEditsTranslateCaptionsUnlocked()
-                translateImageCaptionsTask.disabled = !Prefs.isSuggestedEditsTranslateCaptionsUnlocked()
             }
 
             if (WikipediaApp.getInstance().language().appLanguageCodes.size < MIN_LANGUAGES_TO_UNLOCK_TRANSLATION && Prefs.showSuggestedEditsMultilingualTeaserTask()) {
                 displayedTasks.add(multilingualTeaserTask)
-                multilingualTeaserTask.showActionLayout = Prefs.isSuggestedEditsTranslateDescriptionsUnlocked()
```
Comment (cooltey) [inline app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt]: Should set this `multilingualTeaserTask.showActionLayout = true`?

**Comment by dbrant** [comment]
👍

**Pull request merged**